src/Compilers/Core/Portable/InternalUtilities/PerformanceSensitiveAttribute.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace Roslyn.Utilities
+{
+    /// <summary>
+    /// Indicates that a code element is performance sensitive under a known scenario.
+    /// </summary>
+    /// <remarks>
+    /// <para>When applying this attribute, only explicitly set the values for properties specifically indicated by the
+    /// test/measurement technique described in the associated <see cref="Uri"/>.</para>
+    /// </remarks>
+    [Conditional("EMIT_CODE_ANALYSIS_ATTRIBUTES")]
+    [AttributeUsage(AttributeTargets.Constructor | AttributeTargets.Method | AttributeTargets.Property | AttributeTargets.Field, AllowMultiple = true, Inherited = false)]
+    internal sealed class PerformanceSensitiveAttribute : Attribute
```

Comment:
>PerformanceSensitiveAttribute [](start = 26, length = 29)

It looks like this file already exists in the repo at that same location. Is it just a case of concurrent PRs?

Reply:
➡️ It's the same commit. If we want to wait for a new set of builds I can tell the PR to update itself and this addition will disappear from the diff.

Reply:
Ok, that's fine as long as you merge and don't squash ;-)